tests/frequentist_stats/helpers.py

```diff
@@ -21,7 +21,6 @@ def __init__(self):
         self.target = "search_count"
         self.formula = "search_count ~ C(branch, Treatment(reference='treatment-a'))"
         self.results = smf.ols(self.formula, self.model_df).fit()
-        self.alphastreatment_branches = ["control", "treatment-b"]
```

Comment:
Is this related to the PR or just some cleaning up of an unused var?

Reply:
Not related. I'm not sure where this came from, so I was trying to clean up an unused var.